opentech/settings/base.py

```diff
@@ -674,3 +674,38 @@
 PROJECTS_ENABLED = False
 if env.get('PROJECTS_ENABLED', 'false').lower().strip() == 'true':
     PROJECTS_ENABLED = True
+
+
+# Debug Toolbar
+# https://django-debug-toolbar.readthedocs.io/en/latest/index.html
+if DEBUG:
+    INSTALLED_APPS = [
+        *INSTALLED_APPS,
+        'debug_toolbar',
+    ]
+    MIDDLEWARE = [
+        'debug_toolbar.middleware.DebugToolbarMiddleware',
+        *MIDDLEWARE,
+    ]
+    INTERNAL_IPS = ['127.0.0.1', '10.0.2.2']
```

Comment:
Not strictly sure this is necessary anymore as debug toolbar would handle all that internal ips does.

Reply:
`INTERNAL_IPS` is a [debug toolbar setting](https://django-debug-toolbar.readthedocs.io/en/latest/installation.html#configuring-internal-ips) 🙂

Reply:
Ah sweet, did not know it was used there as well! Assumed it was from django core.

Can we add a comment about `10.0.2.2` being for the vagrant box then.

Reply:
Yea, it's a poorly named setting!

That's a good idea, if there's a more in-depth docs page I'll link to that too.